zqd/pcapstorage/pcapstorage.go

```diff
@@ -0,0 +1,210 @@
+package pcapstorage
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"sync"
+
+	"github.com/brimsec/zq/pcap"
+	"github.com/brimsec/zq/pcap/pcapio"
+	"github.com/brimsec/zq/pkg/iosrc"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zqd/api"
+	"github.com/brimsec/zq/zqe"
+)
+
+const (
+	MetaFile = "pcap.meta.json"
+)
+
+type Store struct {
+	meta meta
+	root iosrc.URI
+	mu   sync.Mutex
+}
+
+type meta struct {
+	PcapURI iosrc.URI
+	Span    nano.Span
+	Index   pcap.Index
+}
+
+type Info struct {
+	PcapURI  iosrc.URI
+	PcapSize int64
+	Span     nano.Span
+}
+
+func New(root iosrc.URI) *Store {
+	return &Store{root: root}
+}
+
+func Open(u iosrc.URI) (*Store, error) {
```

Comment:
Maybe "Load" is a better term since there isn't now & probably won't in the future be a need for a "close" like operation?

```suggestion
func Load(u iosrc.URI) (*Store, error) {
```